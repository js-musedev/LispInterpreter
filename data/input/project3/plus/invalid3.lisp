(PLUS 23 (CONS 34 NIL))