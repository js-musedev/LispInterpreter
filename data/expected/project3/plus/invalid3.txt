Error! Parameter at position: 2 of function PLUS is not atomic!    Actual: (34 . 20)
